doc/paper/paper.tex

```diff
@@ -0,0 +1,1012 @@
+    in context} is implemented using a complex visualization application such as
+    Foxglove.
+
+    Beyond these parallels, there are other points of divergence:
+
+    \textbf{Different organizational profile.} Server application logs are
+    generally browsed only by infrastructure teams and application
+    administrators. The read activity on those databases is sparse and high
+    latencies are tolerable. In a robotics development organization, a
+    multimodal log database is a business-critical, user-facing service and
+    may be depended on by most of the development team outside of
+    infrastructure. High latencies in playback and search are a direct
+    impediment to product development speed.
+
+    \textbf{Different scales of volume.} Depending on what kind of data the user
+    is logging, multimodal data volumes can be orders of magnitude larger
+    than server application log volumes for the same organization. Large
+    producers may produce hundreds of terabytes or petabytes of logs per day.
+    This makes it relatively more important for administrators to have a
+    clear understanding of data access patterns, and granular control over
+    data retention.
+
+    \textbf{Support for distributed compute.} Typical server log databases do not
+    expect you to run distributed compute jobs on your log data and are not
+    designed with that use-case in mind, whereas in multimodal log management
+    it is a core workload.
+
+    \textbf{Support for statistical aggregates.} Multimodal logs are logs, but
+    in a lot of cases they are also similar to metrics. For some kinds of logs
+    it is attractive to be able to plot statistical aggregates such as
+    quantiles or averages of field values over selected windows of time. The
+    APM portions of Datadog enable this kind of operation at low latency over
+    trillions or quadrillions of points. To drive a responsive interface these
+    aggregations must rely on precomputation.
+
+    Of course, it can be assumed that in some large organizations these concerns
+    all apply to server logs as well. Many organizations employ more infrastructure
+    engineers than all the employees in the largest robotics company, and some of
+    them undoubtedly produce petabytes of logs and analyze them with Kafka
+    \cite{kafka} or Spark \cite{spark}.
+
+    However, in multimodal logging these scales are hit by startup companies, and
+    the scaling requirements are significantly a function of the product’s
+    properties and requirements – i.e what kinds of sensors and how good the
+    performance needs to be – while in contrast the scaling requirements of
+    traditional log databases are generally driven by organizational scale – i.e
+    how many thousands of internal microservices there are. Organizational scale is
+    slower-growing because it usually requires some form of sustained business
+    success over years.
```

Comment:
I feel that there are some untestable hypotheses in here. Could we shorten this to simply state the challenge(s) brought on by large scales of multimodal log data